homeassistant/components/sense/__init__.py

```diff
@@ -69,16 +75,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     gateway.rate_limit = ACTIVE_UPDATE_RATE
 
     try:
-        await gateway.authenticate(email, password)
-    except SenseAuthenticationException:
-        _LOGGER.error("Could not authenticate with sense server")
-        return False
-    except SENSE_TIMEOUT_EXCEPTIONS as err:
-        raise ConfigEntryNotReady(
-            str(err) or "Timed out during authentication"
-        ) from err
-    except SENSE_EXCEPTIONS as err:
-        raise ConfigEntryNotReady(str(err) or "Error during authentication") from err
+        gateway.load_auth(access_token, user_id, monitor_id)
```

Comment:
Note to self: check if this is async safe

Reply:
The `load_auth` function just sets variables, it does not contain any blocking code